### PR TITLE
[v0.16.x] Backport remaining PRs

### PR DESCRIPTION
**Pull request by xdustinface**

### CODE REVIEW

File: src/qt/guiutil.cpp [hunk 1]
```diff
@@ -310,8 +287,6 @@ void setupAddressWidget(QValidatedLineEdit *widget, QWidget *parent, bool fAllow
 {
     parent->setFocusProxy(widget);
 
-    setFixedPitchFont({widget});
-#if QT_VERSION >= 0x040700
```
Comment (UdjinM6) [inline src/qt/guiutil.cpp:314]: should revert `#if` removal

**Review by UdjinM6 — APPROVED** [review]
ACK

**Review by PastaPastaPasta — DISMISSED** [review]
utACK

**Comment by PastaPastaPasta** [comment]
Go ahead and update 3746 commit for the changes in the PR

**Review by UdjinM6 — APPROVED** [review]
re-ACK

**Review by PastaPastaPasta — APPROVED** [review]
utACK, builds, test suite passes

**Comment by UdjinM6** [comment]
FYI: removed "backport candidate" labels from corresponding PRs.

**Pull request merged**